src/DotNetty.Transport/Channels/AbstractChannel.cs

```diff
@@ -366,7 +366,7 @@ internal IMessageSizeEstimatorHandle EstimatorHandle
         /// <summary>
         /// {@link Unsafe} implementation which sub-classes must extend and use.
         /// </summary>
-        protected abstract class AbstractUnsafe : IChannelUnsafe
+        public abstract class AbstractUnsafe : IChannelUnsafe
```

Comment:
Yeah, whoops - sorry about that. Got confused - was reading this on mobile from the airport bar when I wrote this comment originally. Had to make the modifier `public` so it could be subclassed as part of `EmebddedChannel` for `DefaultUnsafe`. What I wrote initially is correct.